src/sentry/static/sentry/app/views/organizationIntegrations/pluginDetailedView.tsx

```diff
@@ -305,6 +313,18 @@ const DisabledNotice = styled(({reason, ...p}: {reason: React.ReactNode}) => (
   font-size: 0.9em;
 `;
 
+const StatusWrapper = styled('div')`
+  margin-bottom: ${space(1)};
+  padding-left: ${space(2)};
+  line-height: 1.5em;
+`;
+
+const Status = styled(p => (
+  <StatusWrapper>
+    <IntegrationStatus {...p} />
+  </StatusWrapper>
+))``;
+
```

Comment:
Are these components identical to the ones in `integrationDetailedView`? Cuz if so, you might wanna try abstracting it a bit and reusing them (imo)

Reply:
@leeandher Yes, I am planning on refactoring it when I refactor all the `detailed views` into one component. But I think this is a style that is specific to the `detailed views` and should not be included in the IntegrationStatus component.

Reply:
@leeandher I made a similar comment. We will do that in a follow-up PR that does refactoring.